### PR TITLE
Style Fixes for _comment.html.erb Partial

### PR DESCRIPTION
**Pull request by noi5e**
- Removes the weird grey bar that Jeffrey noticed in [this comment](https://github.com/publiclab/plots2/issues/8775#issuecomment-762424107)
  - Fixed by assigning unique ID to edit comment form wrapper, and toggling on **that**
- Removes extra `padding-top` and `padding-top` around emoji reaction buttons (also mentioned in that comment)
- Increase margin space between comments
- Add tooltips for Edit Comment and Emoji Reaction
- Add `user-select: none;` to `Reply to comment` link (makes it un-highlight-able by the user)
- Change hover style on emojis
- Opening Edit Comment form now toggles to hide emoji reactions bar

Will leave detailed notes in the code review, and screenshots!

---
(This issue is part of the larger Comment Editor Overhaul Project with Outreachy. Refer to Planning Issue #9069 for more context)

### CODE REVIEW

File: app/views/comments/_edit.html.erb [hunk 1]
```diff
@@ -1,146 +1,152 @@
-<div class="comment-form-wrapper">
-  <div class="card card-body bg-light">
-    <form method="post" id="c<%= comment.id %>edit" style="display:none;" class="well" <% if comment.is_a?Answer %> action= "/answers/update/<%= comment.id %>" <% else %> action="/comment/update/<%= comment.id %>" <% end %>>
+<div id="edit-comment-form-wrapper-<%= comment.id %>" class="card card-body bg-light comment-form-wrapper" style="display: none;">
+  <form method="post" id="c<%= comment.id %>edit" style="display:none;" class="well" <% if comment.is_a?Answer %> action= "/answers/update/<%= comment.id %>" <% else %> action="/comment/update/<%= comment.id %>" <% end %>>
```
Comment (noi5e) [inline app/views/comments/_edit.html.erb:2]: This is the fix for the weird superfluous grey bar mentioned [here](https://github.com/publiclab/plots2/issues/8775#issuecomment-762424107). The edit comment form, when opened, was only toggling on `<form>`, not `<form>`'s container.

**Before:** (grey bar below comment)
Screen Shot 2021-01-25 at 9 16 25 PM

**After:** (no grey bar)
Screen Shot 2021-01-25 at 9 17 51 PM

File: app/views/notes/_comment.html.erb [hunk 2]
```diff
@@ -67,17 +81,16 @@
...
         </a>
       <% end %>
       <% if current_user %>
-        <button style="border: 1px solid;padding: 4px 6px;" class="btn btn-outline-secondary dropdown" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-          <i class='icon fa fa-smile-o'></i>
-          <sup style="font-size: 12px;">+</sup>
-        </button>    
+        <a aria-label="Leave an Emoji Reaction" rel="tooltip" title="Leave an Emoji Reaction" class="btn btn-outline-secondary btn-sm dropdown" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          <i class='far fa-heart'></i>
+        </a>
```
Comment (noi5e) [inline app/views/notes/_comment.html.erb:91]: I changed the icon for emoji reactions:

**Before:** (button slightly bigger than the other buttons, two icons squashed inside button, smiley face)
Screen Shot 2021-01-25 at 9 22 36 PM

**After:** (button same size as other buttons, single heart icon)
Screen Shot 2021-01-25 at 9 20 06 PM

File: app/views/comments/_edit.html.erb [hunk 3]
```diff
@@ -1,146 +1,152 @@
...
+            $('#c<%= comment.id %>edit .preview-btn').button(this.previewing ? 'previewing' : 'reset');
+            $E.generate_preview('c<%= comment.id %>preview',$('#c<%= comment.id %>text').val())">
+          Preview
+        </a>
+        <a 
+          class="btn btn-default" 
+          onClick="
+            $('#edit-comment-form-wrapper-<%= comment.id %>').toggle();
+            $('#<%= comment.id %>-like-emojis').toggle();
+            $('#c<%= comment.id %>edit').toggle();
```
Comment (noi5e) [inline app/views/comments/_edit.html.erb:134]: I also made a change here to toggle the comment's emoji reactions when opening an Edit Comment form. I think it's a lot less confusing.

**Before**: (emoji heart reaction is still present):
https://user-images.githubusercontent.com/4361605/105804161-f1e25b80-5f53-11eb-97c7-cccbba252114.mov

**After**:
https://user-images.githubusercontent.com/4361605/105804185-00307780-5f54-11eb-98f7-db9e6540767e.mov

**Review by jywarren — APPROVED** [review]
Looks great!!!! Thanks @noi5e for all this fine tuning! It's really starting to shine!

**Pull request merged**